lib/solr/request/runner.rb

```diff
@@ -38,11 +39,12 @@ def call
             raise Solr::Errors::SolrQueryError, solr_response.error_message unless solr_response.ok?
             return solr_response
           rescue Faraday::ConnectionFailed, Faraday::TimeoutError, Errno::EADDRNOTAVAIL => e
+            solr_url_errors[node_url] = e.message
```

Comment:
Isn't error class missing with `message`?

Reply:
I thought `message` is verbose enough, but I'll add error class too